src/components/Nav.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import './Nav.css';
+
+export const Nav = () => {
+  return (
+    <nav className="container-nav">
+      <NavLink
+        className={'nav-link'}
```

Comment:
I think the brackets are extra: `className='nav-link'`

Reply:
ready fixed camila! [2778cd5583ce11d8c3b3f2df0e1b8c2809a3332a](https://github.com/the-collab-lab/tcl-39-smart-shopping-list/commit/2778cd5583ce11d8c3b3f2df0e1b8c2809a3332a)